entries/g/glovoapp.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Glovo": {
+    "domain": "glovoapp.com",
+    "url": "https://glovoapp.com/",

```

Comment:
A `url`-element isn't needed for Glovo since `https://glovoapp.com/` is the default value.
```suggestion
```